solvexia_sdk/process/process.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+import requests
+import json
+import sys
+from solvexia_sdk import api
+
+class process:
+    def __init__(self, processId):
+        self.processId = processId
+    
+    def getProcessList(self, name=None, dateCreatedStart=None, dateCreatedEnd=None):
+        params = {
+            'name': name,
+            'dateCreatedStarted': dateCreatedStart,
+            'dateCreatedEnd': dateCreatedEnd
+        }
+        response = requests.get(api.baseUrl + "processes", params=params, headers=api.access_token)
+        api.statusCodeCheck(response, "Error getting process list")
+        return response.json()
+    
+    def getProcess(self):
+        response = api.apiGet(f"processes/{self.processId}")
+        api.statusCodeCheck(response, "Error getting chosen process")
+        return response.json()
+
+    def createProcessRun(self, optionalName=None):
```

Comment:
Please note that we are allowing update to description for createProcessRun api. This enhancement is currently in QA.